explainer.md

```diff
@@ -336,6 +325,62 @@ if(frameOfRef.bounds) {
 }
 ```
 
+### Mirroring
+
+On desktop devices, or any device which has an external display connected to it, it's frequently desirable to show what the user in the headset is seeing on the exernal display. This is usually referred to as mirroring.
+
+In order to mirror WebVR content to the page, developers specify an `outputCanvas` in the `VRSessionCreateParameters` of an exclusive session. The canvas supplied must have a [`ImageBitmapRenderingContext`](https://developer.mozilla.org/en-US/docs/Web/API/ImageBitmapRenderingContext) or the promise returned by `requestSession` will be rejected. Once the session has started any content displayed on the headset will then be mirrored into that canvas.
+
+When mirroring only one eye's content will be shown, and it should be shown undistorted. The UA may choose to crop the image shown, display it at a lower resolution than originally rendered, and the mirror may be multiple frames behind the image shown in the headset. The mirror may include or exclude elements added by the underlying VR system (such as visualizations of room boundaries) at the UA's discretion. Pages should not rely on a particular timing or presentation of mirrored content, it's really just for the benefit of bystanders or demo operators.
+
+On devices without an external display to mirror to, such as mobile or all-in-one systems, the `outputCanvas` will be ignored.
```

Comment:
Are you operating under the assumption that external display == need to mirror?  What if a user agent/OS is already doing its own mirroring (such as the Windows Mixed Reality Portal)?  It seems like a better end user experience might come from not mirroring a second time.  It might be a bit heavy handed to say that a UA can only ignore the `outputCanvas` if it doesn't have an external display.

Of course with the current design the page has no way of distinguishing this situation.  We'd be likely to see pages always supplying an outputCanvas and then there might be a big gaping hole in the middle of the page in some UA.  Perhaps it's worth indicating the need for mirroring (as a attribute off an object returned by calls to supportsSession?)  There is still the risk that developers will ignore it though.  Hrmmmm....